swift/src/Ice/OutputStream.swift

```diff
@@ -159,13 +161,11 @@ public final class OutputStream {
     }
 
     /// Marks the start of a new slice for a class instance or user exception.
-    ///
-    /// - parameter typeId: `String` - The Slice type ID corresponding to this slice.
-    ///
-    /// - parameter compactId: `Int32` - The Slice compact type ID corresponding to this
-    ///   slice or -1 if no compact ID is defined for the type ID.
-    ///
-    /// - parameter last: `Bool` - True if this is the last slice, false otherwise.
+    /// - Parameters:
+    ///   - typeId: The Slice type ID corresponding to this slice.
+    ///   - compactId: The Slice compact type ID corresponding to this
+    ///     slice or -1 if no compact ID is defined for the type ID.
```

Comment:
Can't we use 120 line length like in the rest of the code?

Reply:
Yes, we should.